app/download.go

```diff
@@ -27,7 +27,7 @@ func (a *App) DownloadFromURL(downloadURL string) ([]byte, error) {
 }
 
 func (s *Server) downloadFromURL(downloadURL string) ([]byte, error) {
-	if !model.IsValidHttpUrl(downloadURL) {
+	if !model.IsValidHTTPUrl(downloadURL) {
```

Comment:
Should this become `IsValidHTTPURL`?

Reply:
I'm incident to say no as https://mattermost.atlassian.net/browse/MM-37772 covers that.

Reply:
That works as well :+1: